[2017-07-17 10:19:46.077] [ERROR] dateFileLog - C:\NB\tm.FGEMS\bizServer\biz\auto\simFeeback.js:42
                var MSatTime=  yjDateTime.toUTC(new Date()),79,P,P] ;//1500257820000
                                                            ^^
SyntaxError: Unexpected number
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.foundFile (C:\NB\yujiang.Foil.Node\src\yjApp.js:192:7)
[2017-07-17 10:40:00.250] [ERROR] dateFileLog - ReferenceError: paramIniTenMinutes is not defined
    at success (C:\NB\tm.FGEMS\bizServer\biz\auto\simFeeback.js:40:31)
    at Object.exports.safeSuccess (C:\NB\yujiang.Foil.Node\src\yjError.js:98:13)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:154:12)
    at Query._callback (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:202:38)
    at Query.Sequence.end (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query._handleFinalResultPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:143:8)
    at Query.EofPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:127:8)
    at Protocol._parsePacket (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (C:\NB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
[2017-07-17 10:41:00.228] [ERROR] dateFileLog - ReferenceError: paramIniTenMinutes is not defined
    at success (C:\NB\tm.FGEMS\bizServer\biz\auto\simFeeback.js:40:31)
    at Object.exports.safeSuccess (C:\NB\yujiang.Foil.Node\src\yjError.js:98:13)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:154:12)
    at Query._callback (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:202:38)
    at Query.Sequence.end (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query._handleFinalResultPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:143:8)
    at Query.EofPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:127:8)
    at Protocol._parsePacket (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (C:\NB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
[2017-07-17 16:28:02.910] [ERROR] dateFileLog - C:\NB\tm.FGEMS\bizServer\biz\auto\simFeeback.js:25
    console.log("fffffff",sql);
           ^
SyntaxError: Unexpected token .
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.foundFile (C:\NB\yujiang.Foil.Node\src\yjApp.js:192:7)
[2017-07-17 16:29:05.107] [ERROR] dateFileLog - ReferenceError: sql is not defined
    at error (C:\NB\tm.FGEMS\bizServer\biz\auto\simFeeback.js:31:33)
    at Object.exports.safeError (C:\NB\yujiang.Foil.Node\src\yjError.js:117:10)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:152:12)
    at Query._callback (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:143:35)
    at Query.Sequence.end (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query.ErrorPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:93:8)
    at Protocol._parsePacket (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (C:\NB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (C:\NB\node_modules\mysql\lib\Connection.js:82:28)
[2017-07-17 16:29:27.545] [ERROR] dateFileLog - C:\NB\tm.FGEMS\bizServer\biz\auto\simFeeback.js:23
    var sql:"delete from FeebackUsage fbu where 1=1 and left(fbu.SatTime,7)<?",
        ^^^
SyntaxError: Unexpected identifier
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.foundFile (C:\NB\yujiang.Foil.Node\src\yjApp.js:192:7)
[2017-07-17 16:31:09.578] [ERROR] dateFileLog - ReferenceError: r is not defined
    at error (C:\NB\tm.FGEMS\bizServer\biz\auto\simFeeback.js:31:22)
    at Object.exports.safeError (C:\NB\yujiang.Foil.Node\src\yjError.js:117:10)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:152:12)
    at Query._callback (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:143:35)
    at Query.Sequence.end (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query.ErrorPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:93:8)
    at Protocol._parsePacket (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (C:\NB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (C:\NB\node_modules\mysql\lib\Connection.js:82:28)
